k8s/deploy_utils.py

```diff
@@ -30,19 +30,22 @@ def render_template(config, template_name, app):
     template = tenv.get_template(template_name)
     config = todict(config)
 
-    config['kubernetes'].update(**config['kubernetes']['apps'][app])
+    config["kubernetes"].update(**config["kubernetes"]["apps"][app])
     return template.render(config)
 
 
 def k8s_apply(ctx, template_text, apply):
-    namespace = ctx.config['kubernetes']['namespace']
-    with tempfile.NamedTemporaryFile(prefix='k8s', suffix='.yaml', delete=False) as f:
-        f.write(template_text.encode('utf-8'))
-        f.write("\n".encode('utf-8'))
+    namespace = ctx.config["kubernetes"]["namespace"]
+    with tempfile.NamedTemporaryFile(prefix="k8s", suffix=".yaml", delete=False) as f:
+        f.write(template_text.encode("utf-8"))
+        f.write("\n".encode("utf-8"))
         f.flush()
 
     print("Rendering template to:", f.name)
-    cmd = '{} apply -n {} -f {}'.format(get_kubectl(), namespace, f.name)
+    if record:
```

Comment:
Where does this variable come from? Did you mean to add it as a parameter to this funciton?

Reply:
Hm, my bad. Thanks for catching this!